.github/workflows/js-tests.yml

```
@@ -0,0 +1,42 @@
+name: JS Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
```

@tofumatt Let's also run this on push, see other comments on #2537:
```suggestion
on:
  push:
    branches:
      - develop
      - master
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
      - ready_for_review
```